shared/src/constants/v2Deployments.json

```diff
@@ -46,6 +46,7 @@
     "RibbonThetaVaultSTETHLogic": "0x51a5a8f2487Ddd446672868D95347B7cBe041f8b",
     "RibbonThetaVaultRETHCall": "0xA1Da0580FA96129E753D736a5901C31Df5eC5edf",
     "RibbonThetaVaultWBTCCall": "0x65a833afDc250D9d38f8CD9bC2B1E3132dB13B2F",
+    "RibbonThetaVaultUNICall": "0xDD9d1B7dEaB1A843A1B584d2CA5903B8A4735deF",
```

Comment:
Think we're missing a few constants here:
`"OptionsPremiumPricerAAVE": "0x8B5ae818a1b892cC560B522aE6139E38D140EE3d"`,
`"StrikeSelectionAAVE": ""` (TBC, going to update this. Will update here with latest address once deployed)

Reply:
i believe we dont use these values in the app too, sorry will remove them